src/installer/installer.go

```diff
@@ -90,6 +85,11 @@ func (i *installer) InstallNode() error {
 		i.log.Errorf("Failed to write image to disk %s", err)
 		return err
```

Comment:
don't know if it's relevant but no one will wait for anything if you return an error

Reply:
So what should happen if we exit here, or anywhere else, after starting the goroutine?
Do we expect the goroution to be canceled?
Maybe after starting the goroutine we should add a `defer` on the context canceltation.
Here is an example on how to cancel errgroups: https://gist.github.com/pteich/c0bb58b0b7c8af7cc6a689dd0d3d26ef